examples/pingone-neo-getting-started/main.tf

```diff
@@ -0,0 +1,425 @@
+resource "pingone_verify_policy" "verify_government_id_policy" {
+  environment_id = pingone_environment.my_environment.id
+  name           = "Example Government ID Verification Policy"
+  description    = "An example policy configured to require verification by government ID and proof of ownership using facial comparison and liveness checks."
+
+  government_id = {
+    verify = "REQUIRED"
+  }
+
+  facial_comparison = {
+    verify    = "REQUIRED"
+    threshold = "HIGH"
+  }
+
+  liveness = {
+    verify    = "REQUIRED"
+    threshold = "HIGH"
+  }
+
+  transaction = {
+    timeout = {
+      duration  = "30"
+      time_unit = "MINUTES"
+    }
+    data_collection = {
+      timeout = {
+        duration  = "15"
+        time_unit = "MINUTES"
+      }
+    }
+    data_collection_only = false
+  }
+}
+
+# PingOne Verify Exampple POlicy: Phone and Email Verification Only
+resource "pingone_verify_policy" "verify_device_policy" {
+  environment_id = pingone_environment.my_environment.id
+  name           = "Example Email and Mobile Phone Number Verification Policy"
+  description    = "An example policy configured to require verification of an email address and mobile phone number using a one-time password."
+
+  email = {
+    verify = "REQUIRED"
+    otp = {
+      attempts = {
+        count = "3"
+      }
+      lifetime = {
+        duration  = "10"
+        time_unit = "MINUTES"
+      }
+      deliveries = {
+        count = 3
+        cooldown = {
+          duration  = "30"
+          time_unit = "SECONDS"
+        }
+      }
+    }
+  }
+
+  phone = {
+    verify = "REQUIRED"
+    create_mfa_device : true
+    otp = {
+      attempts = {
+        count = "3"
+      }
+      lifetime = {
+        duration  = "5"
+        time_unit = "MINUTES"
+      },
+      deliveries = {
+        count = 3
+        cooldown = {
+          duration  = "30"
+          time_unit = "SECONDS"
+        }
+      }
+    }
+  }
+
+  transaction = {
+    timeout = {
+      duration  = "30"
+      time_unit = "MINUTES"
+    }
+    data_collection = {
+      timeout = {
+        duration  = "15"
+        time_unit = "MINUTES"
+      }
+    }
+    data_collection_only = false
+  }
+}
+
+######################################################################################
+#
+# PingOne Credentials
+#
+######################################################################################
+
+## assign an administrator from parent environment Identity_Data_Admin role in the
+## new environment due to P1Creds service role assignment limitation
+data "pingone_role" "identity_data_admin" {
+  name = "Identity Data Admin"
+}
+
+data "pingone_environment" "primary_admin_env" {
+  environment_id = var.pingone_admin_env
+}
+
+data "pingone_user" "primary_admin_user" {
+  environment_id = data.pingone_environment.primary_admin_env.environment_id
+  email          = var.pingone_admin_email
+}
+
+resource "pingone_role_assignment_user" "primary_admin_role_assignment" {
```

Comment:
As a configuration reference, I think this should be left out.  It might be better to add a README.md link to https://terraform.pingidentity.com/examples/pingone/role-assignment/

Reply:
Will do.  I threw it in at the last minute because I was concerned that someone runs the configuration, then goes to P1 Credentials and sees the odd behavior (you can see the Credentials link, but when you try to view anything, you don't have permissions).  But I agree that this adds unnecessary complexity to the reference - I'l remove and clarify somewhere in the documentation.